### ISSUE TITLE
Difference between m49 and comtrade codes

### ISSUE THREAD

**Issue by sebastian-c** [SWS-Methodology/faoswsTrade#12]
Some countries such as France have different M49 codes (France 250) to comtrade M49 codes (France 251).

Other countries affected by this are Italy, France and the United States. Currently, [we use the fs2m49 function to convert to M49](https://github.com/SWS-Methodology/faoswsTrade/blob/2e5aa77489e578cd26f7c10ebab53aa3ab890d29/modules/complete_tf_cpc/main.R#L593), but we should be using a different table to convert to comtrade M49.

The results of this incorrect mapping is that these countries do not appear in the final result.

**Comment by marcosmilzo** [comment]
At the moment, the data coming from tariffline, reported in M49 comtrade, are converted firstly in fao country code, and then at the end converted to M49 (not comtrade). 
True double mapping is not necessary, and just mapping from M49 comtrade to M49 might be the best solution.

**Comment by sebastian-c** [comment]
Remind me, why is it necessary to convert to faostat codes at all? It looks to me like the destination table uses comtrade M49 codes. Here are the key countries and the codes for them:

```
> GetCodeList("trade", "completed_tf_cpc", "partnerCountryM49")[grepl("France|US[ A]|Italy", description),]
   code              description selectionOnly    type startDate endDate
1:  850           US Virgin Isds         FALSE country        NA      NA
2:  849    US Misc. Pacific Isds         FALSE   group        NA      NA
3:  842               USA(1981-)         FALSE country        NA      NA
4:  841 USA (before 1981)(-1981)         FALSE country        NA      NA
5:  381                    Italy         FALSE country        NA      NA
6:  251            France(1996-)         FALSE country        NA      NA
```

**Comment by sebastian-c** [comment]
The current proposal is to make a new table for trade with regular M49 codes. The discussion is on SWS-1343.

**Comment by sebastian-c** [comment]
There's a new set of tables in trade, but some of the names for keys have changed, as well as the tables.

```
> GetCodeList("trade", "completed_tf_cpc_m49", "geographicAreaM49Partner")[grepl("France|US[ A]|Italy", description),]
   code  description selectionOnly    type startDate endDate
1:  381        Italy         FALSE country        NA      NA
2:  850 US Virgin Is         FALSE country        NA      NA
3:  581  US Minor Is         FALSE country        NA      NA
4:  380        Italy         FALSE country        NA      NA
5:  250       France         FALSE country        NA      NA
```

The module will have to be modified to use a different table as well the correct names of keys for those tables.

Here's the new set of trade tables

```
> GetDatasetNames("trade")
[1] "completed_tf_cpc_m49" "completed_tf_fcl_m49" "completed_tf_m49"     "reliability_index"    "total_trade_cpc_m49"  "total_trade_fcl_m49" 
[7] "total_trade_m49"
```

Here's an example set of config for a cpc_m49 table:

```
> GetDatasetConfig("trade", "completed_tf_cpc_m49")
$domain
[1] "trade"

$dimensions
[1] "geographicAreaM49Reporter" "geographicAreaM49Partner"  "measuredElementTrade"      "measuredItemCPC"           "timePointYears"           

$flags
[1] "flagObservationStatus" "flagMethod"           

$timeDimension
[1] "timePointYears"

$officialFlag
NULL

$officialFlagValues
NULL

$calculatedFlag
NULL

$calculatedFlagValues
NULL
```

**Comment by sebastian-c** [comment]
This issue can't be confirmed fixed until we fix #17

**Comment by sebastian-c** [comment]
Write works and the new table uses the new commodity codes
